test/unit/support/helpers.js

```diff
@@ -26,6 +26,22 @@ export function initRecurly (recurly, opts) {
   return recurly;
 }
 
+let describePartCount = 0;
+export function describePart (...args) {
+  const { SUITE_PARTITION } = window.__env__;
+  const partition = mod => SUITE_PARTITION === 'A' ? mod : !mod;
```

Comment:
@chrissrogers Is the execution running the tests in the same order? If not, I think we could be missing some tests.

Reply:
Wondering if will worth to override the mocha `describe` function in one place?

Reply:
I did explore that path for a little while, but dropped it since there isn't a very good built-in way to monkey patch mocha's methods in browser test contexts.

Order is consistent, yes.

Reply:
On the `describe` override as well, you can't control for only partitioning the outermost describe calls. If we took the approach of striping the skips like I have here, then inner describe blocks will stripe along with the outermost. for example:

```js
describe(
  describe(
    describe()
    describe()
  )
);
```

would compose to:

```js
describe.skip(
  describe(
    describe.skip()
    describe()
  )
);
```

which would skip the entire suite instead of partitioning.